src/EvoSC.Manialinks/Templates/Controls/Alert.mt

```diff
@@ -0,0 +1,80 @@
+﻿<!--
+    Shows an animated alert box with a text and icon.
+-->
+<component>
+    <!-- The ID of the control. -->
+    <property type="string" name="id" default="evosc_alert" />
+    
+    <!-- X position of the control. -->
+    <property type="double" name="x" default="-30.0" />
+    
+    <!-- Y position of the control. -->
+    <property type="double" name="y" default="0.0" />
+    
+    <!-- The area width of the alert text. -->
+    <property type="double" name="width" default="60.0" />
+    
+    <!-- Icon to show for the alert. -->
+    <property type="string" name="icon" default="" />
+    
+    <!-- The text to display in the alert. -->
+    <property type="string" name="text" default="This is an alert" />
+    
+    <template>
+        <frame 
+                pos="{{ x }} {{ y }}" 
+                size="{{ width+8 }} 7" 
+                id="{{ id }}" 
+                data-width="{{ width+7 }}"
```

Comment:
Is there a reason why it is `+7` here and `+8` above?

Reply:
It would clip the entire thing by 1 unit if not for some reason.